Development/nmos/ocsp_behaviour.cpp

```diff
@@ -141,9 +142,9 @@ namespace nmos
 
     namespace details
     {
-        web::http::client::http_client_config make_ocsp_client_config(const nmos::settings& settings, load_ca_certificates_handler load_ca_certificates, slog::base_gate& gate)
+        web::http::client::http_client_config make_ocsp_client_config(const nmos::settings& settings, load_ca_certificates_handler load_ca_certificates, bool secure, slog::base_gate& gate)
```

Comment:
Isn't OCSP always done over HTTP, not HTTPS? Hmm...

This discussion explains why using HTTP is the best practice... https://social.technet.microsoft.com/Forums/office/en-US/c65e1784-39be-4732-a135-bfba7446ad05/should-the-ocsp-responder-service-be-running-http-80-or-https-443-

But the spec says it MAY be done over HTTPS... https://datatracker.ietf.org/doc/html/rfc6960#appendix-A.1